spec/forms/candidate_interface/volunteering_role_form_feature_flag_on_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
```

Comment:
This is a straight copy from the existing form spec. No changes have been made in this commit.